quickstart/end-to-end_sales-forecasting/python_scripts/sales_forecasting_1_create_project_dev_preds_FI.py

```diff
@@ -0,0 +1,208 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+import pandas as pd
+import numpy as np
+import pickle
+import random
+import sklearn
+from sklearn.model_selection import TimeSeriesSplit
+from sklearn.linear_model import LogisticRegressionCV
+from sklearn.linear_model import RidgeCV
+from sklearn.preprocessing import OneHotEncoder
+from sklearn.model_selection import train_test_split
+from sklearn.ensemble import RandomForestClassifier
+from truera.client.truera_workspace import TrueraWorkspace
+from truera.client.truera_authentication import TokenAuthentication
+from truera.client.truera_authentication import BasicAuthentication
+from truera.client.ingestion import ColumnSpec, ModelOutputContext
+
+from truera.client.ingestion.util import merge_dataframes_and_create_column_spec
+
+# connection details
+TRUERA_URL = "https://app.truera.net"
+AUTH_TOKEN = "<insert auth token>"
+
+auth = TokenAuthentication(AUTH_TOKEN)
+tru = TrueraWorkspace(TRUERA_URL, auth, ignore_version_mismatch=True)
+
+#create project
+project_name = "Sales Forecasting"
+tru.add_project(project_name, score_type="regression")
+tru.activate_client_setting('create_model_tests_on_split_ingestion')
+
+X_train_pre = pd.read_csv('./pre_train.csv',index_col=0).reset_index()
```

Comment:
can we read data directly from the web source instead of forcing users to download on their own?

Reply:
I can do this in a future iteration, but I don't consider this a blocker for merging, do you?